public/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
remove the react icons, manifest, just remove whole boilerplate created by CRA which we won't need

Reply:
i will leave those icons like they are, because we will be replacing them with our own logo. ok?